packages/@sfx/dom-events-plugin/test/unit/browser/events-browser-plugin.test.ts

```diff
@@ -0,0 +1,138 @@
+import { expect, stub } from '../../utils';
+import DomEventsPlugin from '../../../src/dom-events-plugin';
+
+describe('domEventsPlugin', () => {
+  let domEventsPlugin: any;
+
+  beforeEach(() => {
+    domEventsPlugin = new DomEventsPlugin();
+  });
+
+  describe('metadata', () => {
+    it('should be named dom_events', () => {
+      const pluginName = domEventsPlugin.metadata.name === 'dom_events';
+
+      expect(pluginName).to.be.true;
```

Comment:
Remove the two lines above this as well.

```suggestion
      expect(domEventsPlugin.metadata.name).to.equal('dom_events');
```